### PR TITLE
Merge `add-remove` into `develop` branch

### PR DESCRIPTION
**Pull request by o-rashad**
#### What's new :question: 
- Using Hooks `useDispatch`, react components now can update the Redux store.
- Implement remove and add functionality to the app
- Deployed to Github Pages [Link](https://od-c0d3r.github.io/bookstore-react-redux/)
- Update the docs

### CODE REVIEW

File: src/app/store.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
-const store = configureStore({
-  reducer: {
-    books: bookReducer,
-  },
+const reducers = combineReducers({
+  booksState,
 });
 
+const store = createStore(
```
Comment (dasileker) [inline src/app/store.js]: - [x] This file should be in the redux folder.

config

File: src/features/books/booksReducer.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  books: [{ id: 0, title: 'book1', author: 'Willam' }, { id: 1, title: 'book2', author: 'Omar' }],
+};
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
```
Comment (dasileker) [inline src/features/books/booksReducer.js]: - [x] Also this file should be in the redux folder.

redux-book

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @od-c0d3r,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by o-rashad** [comment]
@dasileker Thanks for the review the changes have been made ! :+1:

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @od-c0d3r, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Please it will be nice to structuring your projects using the `ducks pattern`, and also to be giving descriptive names for your files and directories(e.g. `redux` for `redux` files instead of `feature` )

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by o-rashad** [comment]
@Mihndim2020 Thanks for the review :blush:

**Pull request merged**